NitroxModel/Helper/StringHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Text;
+
+namespace NitroxModel.Helper
+{
+    public static class StringHelper
+    {
+        public static string GenerateRandomString(int size)
+        {
+            StringBuilder builder = new StringBuilder();
+            Random random = new Random();
```

Comment:
It's best to create a `Random` instance once and reuse it so that when this method is called in a loop it will generate new numbers and not repeat itself. See: https://stackoverflow.com/a/3854950/1277156